spec/vcr_cassettes/manageiq/providers/google/cloud_manager/refresher.yml

```diff
@@ -5,7 +5,7 @@ http_interactions:
     uri: https://accounts.google.com/o/oauth2/token
     body:
       encoding: UTF-8
-      string: assertion=eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJpc3MiOiJzZXJ2aWNlLWFjY291bnQtMUBjaXZpbC10dWJlLTExMzMxNC5pYW0uZ3NlcnZpY2VhY2NvdW50LmNvbSIsImF1ZCI6Imh0dHBzOi8vYWNjb3VudHMuZ29vZ2xlLmNvbS9vL29hdXRoMi90b2tlbiIsImV4cCI6MTQ1MDI3NTA1MiwiaWF0IjoxNDUwMjc0OTMyLCJzY29wZSI6Imh0dHBzOi8vd3d3Lmdvb2dsZWFwaXMuY29tL2F1dGgvY29tcHV0ZSBodHRwczovL3d3dy5nb29nbGVhcGlzLmNvbS9hdXRoL2RldnN0b3JhZ2UucmVhZF93cml0ZSBodHRwczovL3d3dy5nb29nbGVhcGlzLmNvbS9hdXRoL25kZXYuY2xvdWRtYW4gaHR0cHM6Ly93d3cuZ29vZ2xlYXBpcy5jb20vYXV0aC9jbG91ZC1wbGF0Zm9ybSJ9.YAULcKUNhBqhoOv9ocRXgh2nQkKn0GgIfEXwqueb72DnMElUT-p7RcS1098MoDSQd5AaflRZYnMFeZhVJ0sZm1PZoAk0UGtjyqqE2hacp3dnmaSaFfg5jTr0fI-6pXdzx9fnPTfR8lLTuuD64lBFb2JWE7qMd3cj6EVZ49O59BNjXhkWNxYkTNGgbLlbC7eEMkP-lF1fLpbKFhcntSocBILtKiOSQgAfL21n5UTiyZTgPdq6ujwVLkvN_QkGyI2oHTxkjISjRM4DKgKK2VGXCfyRiUqZVCqb-cy3lRFDuGKIKuMf6neXf1ZSMvkgTbBSPYwH5fugdLY2dUGfmd3aGw&grant_type=urn%3Aietf%3Aparams%3Aoauth%3Agrant-type%3Ajwt-bearer
+      string: assertion=eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJpc3MiOiJzZXJ2aWNlLWFjY291bnQtMUBjaXZpbC10dWJlLTExMzMxNC5pYW0uZ3NlcnZpY2VhY2NvdW50LmNvbSIsImF1ZCI6Imh0dHBzOi8vYWNjb3VudHMuZ29vZ2xlLmNvbS9vL29hdXRoMi90b2tlbiIsImV4cCI6MTQ1MDI5ODExMCwiaWF0IjoxNDUwMjk3OTkwLCJzY29wZSI6Imh0dHBzOi8vd3d3Lmdvb2dsZWFwaXMuY29tL2F1dGgvY29tcHV0ZSBodHRwczovL3d3dy5nb29nbGVhcGlzLmNvbS9hdXRoL2RldnN0b3JhZ2UucmVhZF93cml0ZSBodHRwczovL3d3dy5nb29nbGVhcGlzLmNvbS9hdXRoL25kZXYuY2xvdWRtYW4gaHR0cHM6Ly93d3cuZ29vZ2xlYXBpcy5jb20vYXV0aC9jbG91ZC1wbGF0Zm9ybSJ9.OMtVmpXSw0Lj0wC_kbGmGyzXSBQshxOmzG7zGqwefYGwUMbero7XMrpB3yFw8L-PxtBHzxXbZ0efL5XULD9YghUvbG2mTjhoNXWvHxGZehdyzNr-vFBpjm06yJuF20K87NLxcvGXf3El0YRuLFoOCjgQfd5nxqd5dpGOG55LqwUKF4khsT2s1LSMWxvcvAi65owlwOKIsySBAKm_KI6fFS8-fI1RwnPr0dLlAhuh2TiA2eUwBzO0Iyl0A7IHVgIDXqTJ6LHusInCG4VW6oJscHYl9VVjx82o9s9OQw4R3geXFRIGZSx5ByPdFmvaK4WJxRfEiyMGhMkmUwWHpOaP3A&grant_type=urn%3Aietf%3Aparams%3Aoauth%3Agrant-type%3Ajwt-bearer
```

Comment:
This is the aforementioned vcr cassette, this way we don't have to actually communicate with a live provider to run tests.